### ISSUE TITLE
Feature Request: Multiplayer: Connect to room automatically/via command line argument

### ISSUE THREAD

**Issue by hitsu4yrkl** [yuzu-emu/yuzu#8927]
Hello awesome yuzu team, quick quality of life multiplayer suggestion:

An option to set a multiplayer room to connect to automatically on startup thru GUI or rather simple as a command line argument, here an example:
if `yuzu.exe -g "some\path\to\game.xci"` just starts my game,
`yuzu.exe -ip <ip> -port <port> -username "<username>" -password "<password>" -g "some\path\to\game.xci"` could start that game but also automatically connect me to a multiplayer room hosted at the ip address `<ip>` and port `<port>`.

I think this could make the process of playing with friends just a bit more convenient (and should be quite simple to implement, correct me if I'm wrong)!!

**Comment by JumpingY** [comment]
https://yuzu-emu.org/help/feature/multiplayer/

**Comment by agentwolf88** [comment]
This would be cool!

**Comment by bao3** [comment]
Yes, thats great and important for Steam Deck , Please someone help do it .

**Comment by Shynixn** [comment]
The yuzu-cmd.exe already contains such a feature. Can we please have this in yuzu.exe as well?
https://github.com/yuzu-emu/yuzu/blob/bbfd9eb428c7b1c811a3a041d62a7d4c94cafa07/src/yuzu_cmd/yuzu.cpp#L72

**Comment by Shynixn** [comment]
Yes please, I still need this feature

**Comment by SaulGoodman1337** [comment]
please add this simple and long overdue feature
